### PR TITLE
Issues/29

### PR DESCRIPTION
**Pull request by N3llz**
@mcquin

### CODE REVIEW

File: PNAS2009/index.html [hunk 1]
```diff
@@ -0,0 +1,144 @@
+<!---
+layout: default
+redirect_from: "/PNAS2009.shtml"
+--->
```
Comment (mcquin) [inline PNAS2009/index.html]: Remove lines 1-4.

File: PNAS2009/index.html [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+	<meta property="og:url" content="http://cellprofiler.org">
+	<meta property="og:image" content="http://cellprofiler.org/images/gallery.png">
+	<meta property="og:description" content="CellProfiler is a free open-source software for measuring and analyzing cell images">
+	<link rel="stylesheet" href="/css/app.css">
+	<!--Google Analytics-->
+	<script>
+		(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+		})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
```
Comment (mcquin) [inline PNAS2009/index.html]: This is hard to read. Can you fix the formatting of this block to look more like https://github.com/CellProfiler/cellprofiler.github.com/blob/master/citations/index.html#L15-L27 ?

File: PNAS2009/index.html (hunk 1 above)
Comment (N3llz) [inline PNAS2009/index.html]: fixed

File: PNAS2009/index.html (hunk 2 above)
Comment (N3llz) [inline PNAS2009/index.html]: fixed...I think. Have a look and let me know

**Pull request merged**